sdk/Lusid.Sdk/Utilities/ApiRetryHandler.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using Polly;
+using RestSharp;
+
+namespace Lusid.Sdk.Utilities
+{
+    /// <summary>
+    /// Class used to define API error retry rules for all API calls.
+    /// </summary>
+    public static class ApiRetryHandler
+    {
+        /// <summary>
+        /// Number of max retry attempts
+        /// </summary>
+        public const int MaxRetryAttempts = 3;
```

Comment:
Should this be configurable by a user?

Reply:
As in should we allow it to be configurable or is it configurable now?

I don't think users should be able to configure this. I thought a const assignment makes the variable immutable.

Reply:
But let me know what you think

Reply:
Then they create a polly policy on their own end. and assign/wrap it with the global configuration. These are not for the client to modify as they only handle internal exceptions, such as the 0 status code ones

Reply:
Ok